multiqc/modules/pangolin/pangolin.py

```diff
@@ -0,0 +1,157 @@
+#! /usr/bin/env python
+
+""" MultiQC module to parse output from Pangolin """
+
+from __future__ import print_function
+from collections import OrderedDict
+import logging
+import csv
+
+from multiqc import config
+from multiqc.plots import beeswarm, table
+from multiqc.modules.base_module import BaseMultiqcModule
+
+# Initialise the logger
+log = logging.getLogger(__name__)
+
+
+class MultiqcModule(BaseMultiqcModule):
+    """Pangolin module"""
+
+    def __init__(self):
+
+        # Initialise the parent module
+        super().__init__(
+            name="Pangolin",
+            anchor="pangolin",
+            href="https://github.com/cov-lineages/pangolin",
+            info="is a software package for assigning SARS-CoV-2 genome sequences to global lineages.",
+        )
+
+        # Find and parse the sample files
+        self.pangolin_data = dict()
+        for f in self.find_log_files("pangolin", filehandles=True):
+            self.parse_pangolin_log(f)
+
```

Comment:
You're missing the sample filtering and stop-point for zero samples here. So MultiQC will _always_ generate a report and a Pangolin section, even if there are no samples found.

This a pretty common mistake, there's a CI test for it and that's what's failing. It's easy to fix though, I'll push the required code now.

Docs: 

* https://multiqc.info/docs/#filtering-by-parsed-sample-names
* https://multiqc.info/docs/#no-files-found